.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/config'
+    ])
+
+    ->exclude([
+        __DIR__ . '/tests/_output',
+        __DIR__ . '/tests/Support/_generated',
+    ])
```

Comment:
Can be removed, since we have no tests.
```suggestion
```

Reply:
But there will be tests, so I would keep them for now.

Reply:
All right.. good to know.